.github/workflows/build.yml

```
@@ -3,15 +3,20 @@ on:
+concurrency:
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
 
 jobs:
   build:
-    if: (github.event_name == 'pull_request' && github.event.action == 'labeled') || github.event_name != 'pull_request' || github.repository != github.event.pull_request.head.repo.full_name
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v6.0.2
       - uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.12)

[error] the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)